pkg/plugins/platform/utils.go

```diff
@@ -128,32 +128,6 @@ func ComposeFileExists(app App) bool {
 	return true
 }
 
-// Cleanup will clean up ddev apps even if the composer file has been deleted.
-func Cleanup(app App) error {
```

Comment:
Do we actually need this extra cleanup function? I don't have all the background surrounding its original introduction, so it's hard for me to really say. I suppose it could be a nice to have in case the user deletes their project without removal, but this could also be a scenario sufficiently addressed with a little bit of troubleshooting docs.